README.md

```diff
@@ -1,73 +1,25 @@
-# Greenpeace Planet4 Master Theme
+# Greenpeace Planet 4 Master Theme
 
 ![Planet4](./images/planet4.png)
 
-## Introduction
+## What is it?
 
-### What is it?
-This is the master Wordpress theme for Greenpeace Planet4 project.
+This is the master Wordpress theme for Greenpeace Planet 4 project.
 You can learn more about this project on [the planet4 blog](https://medium.com/planet4).
```

Comment:
Maybe this should also be updated to `the Planet 4 blog`? (sorry I swear it's the last typo I spot 😬)